frontend/src/metabase/browse/components/BrowseApp.tsx

```diff
@@ -48,6 +52,62 @@ export const BrowseApp = ({
     }
   }, [tab]);
 
+  const availableModelFilters =
+    PLUGIN_CONTENT_VERIFICATION.availableModelFilters;
```

Comment:
this can be outside the function body, right, that way it won't need to be in react dependency arrays